be/src/io/cache_input_stream.h

```diff
@@ -114,6 +118,8 @@ class CacheInputStream final : public SeekableInputStreamWrapper {
     BlockCache* _cache = nullptr;
     int64_t _block_size = 0;
     std::unordered_map<int64_t, BlockBuffer> _block_map;
+    int8_t _priority = 0;
```

Comment:
consider defining a enum for priority in `WriteCacheOption` for readability

Reply:
In star cache, we use `std::vector<LRUContainer>` to hold different priority's lru. The priority is the array index for `std::vector<LRUContainer>`.

And star cache's API is using int8_t as a parameter, so using enum here is unnecessary.

In addition, if many priorities are provided in the future like starburst, it will be difficult to expand the enum here.

If we use `enum` here, it